applications/ContactStructuralMechanicsApplication/symbolic_generation/ALM_frictional_mortar_condition/ALM_frictional_mortar_contact_condition.cpp

```diff
@@ -31,7 +31,7 @@ Condition::Pointer AugmentedLagrangianMethodFrictionalMortarContactCondition<TDi
     NodesArrayType const& rThisNodes,
     PropertiesPointerType pProperties ) const
 {
-    return boost::make_shared< AugmentedLagrangianMethodFrictionalMortarContactCondition<TDim,TNumNodes,TNormalVariation> >( NewId, this->GetGeometry().Create( rThisNodes ), pProperties );
+    return Condition::Pointer( new AugmentedLagrangianMethodFrictionalMortarContactCondition<TDim,TNumNodes,TNormalVariation >( NewId, this->GetGeometry().Create( rThisNodes ), pProperties ));
```

Comment:
can i ask why you switched back to the Pointer construction? the make_shared should be better